src/Pages/Login.css

```
@@ -0,0 +1,83 @@
+.login {
+  width: 100vw;
+  height: 100vh;
+  overflow: hidden;
+  display: flex;
+}
+
+.login > div {
+  flex: 1;
+  min-width: 0;
+  margin: 24px;
+}
+
+.login .logo {
+  margin-top: 16px;
+  margin-bottom: 67px;
+}
+
+.login > div:nth-child(1) {
+  display: flex;
+  justify-content: center;
+}
+
+.login > div:nth-child(1) > .login-container {
+  width: 403px;
+}
+
+.login > div:nth-child(2) > div.artwork {
+  background-image: var(--img-src);
+  width: 100%;
+  height: 100%;
+  border-radius: 50px;
+  background-size: cover;
+  background-position-x: center;
+  display: flex;
+  align-items: flex-end;
+}
+
+.login > div:nth-child(2) > div.artwork > div {
+  margin-left: 25px;
+  margin-bottom: 25px;
+  padding: 4px 12px;
+  background-color: var(--modal-bg-color);
+  border-radius: 1em;
+  color: var(--gray-light);
+  font-size: 14px;
+}
+
+.login > div:nth-child(2) > div.artwork .zap-button {
+  display: inline-block;
+  color: inherit;
+  background-color: inherit;
+}
+
+@media (max-width: 1024px) {
+  .login > div:nth-child(2) {
+    display: none;
+  }
+}
+
+.login input {
+  background-color: var(--gray-secondary);
+  padding: 12px 16px;
+  font-size: 16px;
+}
+
+.login .login-note {
+  color: var(--gray-light);
+  font-size: 14px;
+}
+
+.login .login-actions {
+  margin-top: 24px;
+}
+
+.login .login-actions > button {
+  margin-right: 10px;
+}
+
+.login .login-or {
+  margin-top: 56px;
+  margin-bottom: 64px;
+}
```

I'd add a small margin to the left of the zap button:

```suggestion
}

.login .artwork .zap-button { 
  margin-left: 6px; 
}
```

---

Didnt add margin, but removed the BG color so it looks like there is some spacing instead.